libs/lib-mongodb/package.json

```diff
@@ -31,6 +31,7 @@
     "@powersync/lib-services-framework": "workspace:*",
     "bson": "^6.10.3",
     "mongodb": "^6.14.1",
+    "mongodb-connection-string-url": "^3.0.2",
```

Comment:
We can remove uri-js below now - this was the only place it was used in this package.